docs/.vuepress/components/ColorSwatch.vue

```diff
@@ -108,31 +116,29 @@ export default {
      * @todo Need to handle error here if the colour doesn't exist
      *       in the main list.
      */
-    colorHex: function() {
-      let source =
-        this.theme === 'socrative' ? SOCRATIVE.colors : SHOWBIE.colors;
+    chromaColor: function() {
+      let source = this.theme === 'socrative' ? SOC.colors : SBE.colors;
+      let color = this.hex || source[this.hue][this.scale];
```

Comment:
This fails to grab the "alpha" keys, *i.e.* if `this.scale` is `'900-95'`, but it's not that important right now.

Reply:
NM, works now